README.md

```diff
@@ -45,7 +45,13 @@ This specific project holds the officialy supported Collaboration Tree backend.
 
 ## Technology
 
-The technology is based on NGINX as proxy server, NaviServer as webserver, TCL-TK and Python3 as programing languages, PostgreSQL as database manager system and PSQL as SQL language script. 
+The technology is based on:
+1. Unix-Like Operating System, Debian and Ubuntu are most used, https://www.debian.org/index.pt.html and https://ubuntu.com/;
+2. NGINX as proxy server, https://docs.nginx.com/;
+3. NaviServer as webserver, https://naviserver.sourceforge.io/;
+4. PostgreSQL as database manager system and PSQL as SQL language script. https://www.postgresql.org/docs/ and https://www.postgresql.org/docs/9.2/app-psql.html;
+5. Python3 and TCL-TK as programing languages, https://www.python.org/doc/ and http://tcl.tk/man/tcl8.6/contents.htm;
```

Comment:
Don't need to include ; at the end of each line.  You can check out how it looks in your branch once you push it by going here: https://github.com/F4IF/ctree-backend/tree/iuri-patch-1.  I think you could also improve this by converting links to markup instead of giving the full URL at the end:
`1. Unix-Like Operating System, [Debian](https://www.debian.org/index.pt.html) and [Ubuntu](https://ubuntu.com/) are most used`